src/material/tabs/tab-group.ts

```diff
@@ -88,10 +89,13 @@ interface MatTabGroupBaseHeader {
 // tslint:disable-next-line:class-name
 export abstract class _MatTabGroupBase extends _MatTabGroupMixinBase implements AfterContentInit,
     AfterContentChecked, OnDestroy, CanColor, CanDisableRipple {
-  abstract _tabs: QueryList<MatTab>;
+  abstract _allTabs: QueryList<MatTab>;
```

Comment:
Can you add a JsDoc comment that explains that this may also contain nested tabs-in-tabs?